packages/host/app/components/operator-mode/card-adoption-chain.gts

```diff
@@ -22,16 +23,67 @@ export default class CardAdoptionChain extends Component<Signature> {
       .card-adoption-chain {
         background-color: var(--boxel-200);
       }
+      .content-with-line {
+        position: relative;
+        transform: translateX(calc(var(--boxel-sp-sm) * -1));
+
+        display: flex;
+        align-items: center;
+        width: calc(100% + calc(var(--boxel-sp-sm) * 2));
+        height: 60px;
+        padding: var(--boxel-sp) 0;
+      }
+
+      .inherits-from {
+        display: flex;
+        align-items: center;
+        padding: 0 var(--boxel-sp-xs);
+        gap: var(--boxel-sp-xxxs);
+        font: var(--boxel-font-size-sm);
+        text-wrap: nowrap;
+        background: var(--boxel-200);
+
+        position: absolute;
+        left: 50%;
+        transform: translateX(-50%);
+      }
+
+      .chain:last-child .content-with-line {
+        display: none;
+      }
+
+      .inherits-icon {
+        height: 24px;
+      }
+
+      .line {
+        width: 100%;
+        height: 1px;
+        background-color: var(--boxel-purple-200);
+      }
     </style>
 
     <div class='card-adoption-chain'>
       {{#each @cardInheritanceChain as |data|}}
-        <CardSchemaEditor
-          @card={{data.card}}
-          @cardType={{data.cardType}}
-          @file={{@file}}
-          @moduleSyntax={{this.moduleSyntax}}
-        />
+        <div class='chain'>
+          <CardSchemaEditor
+            @card={{data.card}}
+            @cardType={{data.cardType}}
+            @file={{@file}}
+            @moduleSyntax={{this.moduleSyntax}}
+          />
+          <div class='content-with-line'>
+            <div class='line' />
```

Comment:
This is a `hr` instead of a `div`

Reply:
updated [0e5ba62](https://github.com/cardstack/boxel/pull/707/commits/0e5ba62e49cf4696d7b354ab9e215afe23f14a19)